handbook.md

```diff
@@ -128,6 +128,28 @@ Parental Leave includes the following benefits:
 
 Leave may also be used for medical procedures and bereavement. Speak with your immediate supervisor as soon as you’re aware of the need for extended leave.
 
+## Background Checks
+
+Fishtown Analytics is concerned about the safety of its team members and about maintaining appropriate controls to ensure that assets of GitLab and our customer relationships and information are protected. To reduce these risks, we will obtain and review background information of covered prospective, and, as applicable, current employees.
+
+Fishtown Analytics has contracted with Sterling Talent Solutions to perform these background checks, which will cover criminal history for the last 7 years and employment history for the last 5 years and/or the three most recent employers. Fishtown Analytics may use the returned background check information to make decisions regarding employment. For certain positions where the candidates financial history is relevant to the position, we may also run a check in the federal database for any financial related offenses.
```

Comment:
Is `Sterling Talent Solutions` the org that we're using for this?

Reply:
yep!